.github/workflows/track_minetest.yml

```diff
@@ -0,0 +1,28 @@
+name: track_minetest
+
+# build on c/cpp changes or workflow changes
+on:
+  schedule:
+    # This cron line says to run every minute, but in practice github appears
+    # to run it sporadically every 7 to 15 minutes.
+    - cron: '* * * * *'
```

Comment:
too often is wasteful IMO, how about every 3 hours?

Reply:
Does this use up some kind of resource? The check itself is very fast, only takes about 10 seconds. What takes a while is for github to allocate a VM to run it.

Reply:
There's a limit on concurrent jobs per organization: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration
I don't expect this to be a problem currently but I think it's better to be preemptive.

Reply:
changed to every 3 hours